elements/rh-icon/README.md

```diff
@@ -1,5 +1,53 @@
 # WIP 🐣: RHElements Icon Element
 
+## Usage
+
+```
+<rh-icon icon="rh-icon-server"></rh-icon>
+```
+
+## Attributes
+
+### icon (observed)
+
+The name of the icon that you want to use. If the value of this attribute changes, the new icon will show up in the UI.
+
+### data-size
```

Comment:
I thought we were going to omit `data-` prefixes for custom attributes.

Reply:
We'll fix this when we update the element.